### ISSUE TITLE
Unable to parse interpolation within property

### ISSUE THREAD

**Issue by mahtd** [postcss/postcss-scss#5]
Created this issue from my [comment](https://github.com/postcss/postcss-scss/issues/4#issuecomment-136021699) on the closed issue #4 so that the issue will have more visibility

Expanding the test case for interpolation within a word causes a syntax error. Specifically, it's having a problem with the interpolation on the property (`border-#{$side}-width`). It seems the tokenizer splits this into three separate words, which the parser has a problem with

```js
    it('parses interpolation inside word', () => {
        let root = parse('.#{class}:hover { border-#{$side}-width: #{$width}px; }');
        expect(root.first.selector).to.eql('.#{class}:hover');
        expect(root.first.first.prop).to.eql('border-#{$side}-width');
        expect(root.first.first.value).to.eql('#{$width}px');
    });
```

Error:

```
  1) SCSS Parser parses interpolation inside word:
     CssSyntaxError: <css input>:1:26: Unknown word
  .#{class}:hover { border-#{$side}-width: #{$width}px; }
                           ^
      at Object.scssParse [as default] (lib/scss-parse.es6:10:12)
      at Context.<anonymous> (test/parse.es6:75:20)
```

Tokens:

```js
  [ 'word', 'border-', 1, 19, 1, 25 ],
  [ 'word', '#{$side}', 1, 26, 1, 33 ],
  [ 'word', '-width', 1, 34, 1, 39 ],
```

**Comment by ai** [comment]
Sorry, problem is in current parser. It can uses only one token properties.

I will fix it, but not soon: https://github.com/postcss/postcss/issues/185

**Comment by ai** [comment]
Released in PostCSS 5.0.5.
